code/modules/mob/living/simple_animal/guardian/types/support.dm

```diff
@@ -32,7 +32,10 @@
 		C.adjustBruteLoss(-5)
 		C.adjustFireLoss(-5)
 		C.adjustOxyLoss(-5)
-		C.adjustToxLoss(-5, FALSE, TRUE)
+		if (!HAS_TRAIT(target, TRAIT_TOXINLOVER))
```

Comment:
Instead of this, you can just add "forced = TRUE" to that adjusttoxloss

Reply:
@zeskorion 
```
/mob/living/proc/adjustToxLoss(amount, updating_health = TRUE, forced = FALSE)
	if(!forced && (status_flags & GODMODE))
		return FALSE
	toxloss = CLAMP((toxloss + (amount * CONFIG_GET(number/damage_multiplier))), 0, maxHealth * 2)
	if(updating_health)
		updatehealth()
	return amount
```
How exactly is setting `forced` to `TRUE` inverting the damage for slime people?

Reply:
Setting FORCED=TRUE bypasses the toxinlover trait.

Reply:
Which... we don't want to do. Forced should be false.

Reply:
Ugh, im not home and cant pull it up right now. I'll go find the base code when i am

Reply:
The intent of this pr is to make holoparas not kill slimes, right?

Reply:
```
/mob/living/carbon/adjustToxLoss(amount, updating_health = TRUE, forced = FALSE)
	if(!forced && HAS_TRAIT(src, TRAIT_TOXINLOVER)) //damage becomes healing and healing becomes damage
		amount = -amount
		if(amount > 0)
			blood_volume -= 5*amount
		else
			blood_volume -= amount
	if(HAS_TRAIT(src, TRAIT_TOXIMMUNE)) //Prevents toxin damage, but not healing
		amount = min(amount, 0)
	return ..()
```
It should be false.

Reply:
k